apps/aecontract/src/aect_channel_contract.erl

```diff
@@ -44,8 +44,12 @@ run_new(ContractPubKey, Call, CallData, Round, Trees0) ->
                , trees       => Trees0
                  %% We do not want the execution off chain and
                  %% on chain to be different so the coinbase
-                 %% instruction will allways return 0.
+                 %% instruction will always return 0.
                , beneficiary => <<0:?BENEFICIARY_PUB_BYTES/unit:8>>
+                %% TODO: Default value for offchain
+               , difficulty  => 0
+                %% TODO: The time should probably be set in channels as well
+               , time        => aeu_time:now_in_msecs()
```

Comment:
Since in off-chain execution the calls are executed on each party's side - this could be non-deterministic. I will address it in [PT-160280811_sc_address_on_chain_objects](https://github.com/aeternity/epoch/tree/PT-160280811_sc_address_on_chain_objects) branch